src/lightning/pytorch/loggers/comet.py

```diff
@@ -324,26 +301,21 @@ def log_metrics(self, metrics: Mapping[str, Union[Tensor, float]], step: Optiona
         metrics_without_epoch = _add_prefix(metrics_without_epoch, self._prefix, self.LOGGER_JOIN_CHAR)
         self.experiment.log_metrics(metrics_without_epoch, step=step, epoch=epoch)
```

Comment:
Let's update it to `__internal_api__log_metrics__` and pass `pytorch-lightning` as a source framework.
`__internal_api__xxxx__` is our own naming convention for methods that we don't want to make public in the SDK, because we don't want users to use them.
And at the same time for methods which are not actually private/protected because WE might want to call them in some our code.

The same thing is relevant for `log_parameters` calls here.